### ISSUE TITLE
Search Panes Error in Relationship Table

### ISSUE THREAD

**Issue by FluffyKnightz** [darrachequesne/spring-data-jpa-datatables#159]
I can use search panes with data from relationship entity.

@RequestMapping(value = "/employees-searchpanes-basic", method = RequestMethod.POST)
    public DataTablesOutput<Employee> listWithBasicSearchPanes(@Valid DataTablesInput input, @RequestParam Map<String, String> queryParameters) {
        input.parseSearchPanesFromQueryParams(queryParameters, Arrays.asList("position", "office.city"));
        return employeeRepository.findAll(input);
    }

I want to use office.city in SearchPanes and but when i use like this, 
'DataTables warning: table id=employees - java.lang.IllegalArgumentException: org.hibernate.query.SemanticException: Could not resolve attribute 'office.city' of 'sample.employee.Employee' 
that error happen. How do i fix that?

**Comment by darrachequesne** [comment]
I could indeed reproduce the issue. It seems there is a mistake here: https://github.com/darrachequesne/spring-data-jpa-datatables/blob/be286ee3fd44a106d70cf284f08024cb2e5c0351/src/main/java/org/springframework/data/jpa/datatables/repository/DataTablesRepositoryImpl.java#L112-L113

I think `root.get(attribute)` (here, `root.get("office.city")`) is not valid, the string should be split into `root.get("office").get("city")`. Would you have time to create a PR?

**Comment by FluffyKnight2002** [comment]
i dont know how to do that?

**Comment by darrachequesne** [comment]
@sebastianwehrmann you are right, the counts are displayed properly, but filtering does not work.

**Comment by darrachequesne** [comment]
@sebastianwehrmann this should be fixed by https://github.com/darrachequesne/spring-data-jpa-datatables/commit/cc4b439f2a7b30070c9fa1f4bcb9dab2ceb0bfc0, included in `7.0.1`. Could you please check?

https://github.com/user-attachments/assets/4f310e5d-f2da-4ee8-ace2-244922b45386
